vocabulary-draft/source-code-version-control.md

```diff
@@ -16,21 +16,25 @@ These events are related to Source Code repositories
 
 
 Repository Events MUST include the following attributes:
-- **Event Type**: the type is restricted to include `cd.**` prefix. For example `cd.repository.created` or `cd.repository.changeapproved`
+- **Event Type**: the type is restricted to include `cd.**` prefix. For example `cd.repository.created` or `cd.repository.changeproposalreviewed`
 - **Repository URL**: indicates the location of the source code repository for API operations, this URL needs to include the protocol used to connect to the repository. For example `git://` , `ssh://`, `https://`
 - **Repository Name**: friendly name to list this repository to users
 
 Optional attributes: 
 - **Repository Owner**: indicates who is the owner of the repository, usually a `user` or an `organization`
 - **Repository View URL**: URL to access the repository from a user web browser
 
-From each repository you can emit events related with proposed source code changes. Each change can include a single or multiple commits that can also be tracked. 
+## Change and change proposal events
```

Comment:
Now I am missing something. Why both "Change" and "change proposal"?

Reply:
The idea was that as soon as a change proposal is merged, it stops being a change proposal and becomes an actual change. But based on feedback from you and others, I have chosen to switch to change proposal merged.